client_encryption/encryption_utils.py

```diff
@@ -1,26 +1,47 @@
 from Crypto.PublicKey import RSA
 from Crypto.Hash import SHA1, SHA224, SHA256, SHA384, SHA512
-from OpenSSL.crypto import load_certificate, FILETYPE_PEM, FILETYPE_ASN1, Error
+#from OpenSSL.crypto import load_certificate, FILETYPE_PEM, FILETYPE_ASN1, Error
 from client_encryption.encryption_exception import CertificateError, PrivateKeyError, HashAlgorithmError
+from cryptography import x509
 from cryptography.hazmat.primitives.serialization import pkcs12 
 from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.serialization import Encoding
+from enum import IntEnum
 
 _SUPPORTED_HASH = {"SHA1": SHA1, "SHA224": SHA224, "SHA256": SHA256, "SHA384": SHA384, "SHA512": SHA512}
 
+class FileType(IntEnum):
+    FILETYPE_PEM = 0
+    FILETYPE_ASN1 = 1
+    FILETYPE_INVALID = -1
+
 
 def load_encryption_certificate(certificate_path):
     """Load X509 encryption certificate data at the given file path."""
 
-    try:
+    try: 
         with open(certificate_path, "rb") as cert_content:
             certificate = cert_content.read()
-            x509 = load_certificate(__get_crypto_file_type(certificate), certificate)
-
-        return x509
     except IOError:
-        raise CertificateError("Unable to load certificate.")
-    except (ValueError, Error):
-        raise CertificateError("Wrong encryption certificate format.")
+        raise CertificateError ("Unable to load certificate.")
+    
+    try:         
+        type = __get_crypto_file_type(certificate)
```

Comment:
Don't use `type`, it shadows a builtin function.

Reply:
done